com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -125,6 +172,8 @@ private static ProbeReferenceVolumeAuthoring GetCardinalAuthoringComponent(Probe
 
         private static void OnBakeStarted()
         {
+            FindWorldBounds();
```

Comment:
Hum, so it means that every time we bake, we have to load all the scenes of the project one by one?
I'm afraid this will slow down a lot the baking, especially on big projects :/

Reply:
Couple of points: 

 - This is the only way to avoid re-loading all assets every time a single scene is loaded. That is a potentially huge _runtime_ cost. 
 - This will need to be done on real projects anyway most likely given that any 2 or more scenes that at least share a single cell will need to be baked together to avoid issues. 
 -  We'll  add "bake groups" later on so we don't load _all_ scenes, but only the ones in  the bake group.

Reply:
If this becomes a sensible overhead, we could bake a small asset containing those world bounds info into a separate file (do it once only and update upon scene saved) and use that instead. 

That is a bit overkill though and we have bigger issues now :-) However there are ways when this will become the problem.